### PR TITLE
separate html into individual pages

### PR DESCRIPTION
**Pull request by swanie21**
each page has its own html 

added password and username input fields with a sign-in button

### CODE REVIEW

File: contact-list.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Limelight|Roboto" rel="stylesheet">
+  </head>
+  <body>
+    <section id="contact-list">
+      <h3>Contact List</h3>
+        <form>
+          <input type="text" name="search bar" placeholder="Search Contacts">
+          <button type="button" name="sort button">Sort By</button>
+        </form>
+      <footer id="footer-with-profile">
```
Comment (Jeff-Duke) [inline contact-list.html]: should change this id to a class

File: lib/css/login-screen.scss [hunk 2]
```diff
@@ -1,44 +1,46 @@
-// ***** LOGIN SCREEN *****
-@media (min-width: 320px) {
+@media screen and (min-width: 320px) {
```
Comment (Jeff-Duke) [inline lib/css/login-screen.scss:1]: what is @ media screen and?  i've not seen that before

File: user-profile.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+          <input type="text" name="email" placeholder="primary">
+          <input type="text" name="email" placeholder="secondary">
+        </form>
+        <form class="user-input">
+          <img class="add-button" src="/img/add-button.svg" alt="add button" />
+          <h3>Social:</h3>
+          <input type="text" name="social-media" placeholder="primary">
+        </form>
+      </main>
+      <footer id="home-screen-footer">
```
Comment (Jeff-Duke) [inline user-profile.html]: should either change this to a class or update the ID

**Comment by kylem038** [comment]
:shipit:

**Pull request merged**